### DISCUSSION TITLE
How can i disable view on github button?

### DISCUSSION THREAD

**Discussion by ririko5834** [upptime/upptime#197] [Q&A]
How can I disable  these buttons on my statuspage?
![image](https://user-images.githubusercontent.com/64729293/104120162-ed535b80-5334-11eb-8540-f07b8ba03d96.png)

**Comment by AnandChowdhary** [comment]
This is currently not supported. I'll make sure I add this feature soon!

**Comment by wolfier74** [comment]
I would like this too, i want to keep my github name private also remove posted at date by username when browsing incidents.

Add config option private_mode, when is set to true it should be not showed. And this option should be set to false by default.

**Comment by ririko5834** [comment]
Yes, private_mode config option is good idea. 
Can you please add it? @AnandChowdhary
